Command/DoctrineCommand.php

```diff
@@ -99,6 +99,10 @@ public static function configureMigrations(ContainerInterface $container, Config
             $configuration->setCustomTemplate($container->getParameter('doctrine_migrations.custom_template'));
         }
 
+        if (method_exists($configuration, 'setCheckDatabasePlatform')) {
```

Comment:
Why not bump the constraint to 2.1.0 instead of using such conditionals?

Reply:
It was one of my questions in PR description.

Reply:
Oh right, sorry! Well you know my opinion now.